spec/shr_oncocore.txt

```diff
@@ -0,0 +1,121 @@
+                FindingResult from CancerHistologicTypeVS
+                FindingTopicCode is SCT#371441004 "Histologic type (observable entity)"
+                SpecificFocusOfFinding value is type ref(CancerDisorderPresent)
+0..0			AnatomicalLocation
+0..0            Device
+
+
+EntryElement:   CancerHistologicGrade
+// did not use precoordinated codes because only available for clinical and pathologic.
+Based on:       CodedLaboratoryObservation
+Description:    "A description of a tumor based on how abnormal the cancer cells and tissue look under a microscope and how quickly the cancer cells are likely to grow and spread. Low-grade cancer cells look more like normal cells and tend to grow and spread more slowly than high-grade cancer cells. (Source: NCI Dictionary of Cancer Terms)"
+Concept:        LNC#21858-6 "Grade Cancer"
+                FindingResult from http://loinc.org/vs/LL213-0
+                FindingTopicCode is LNC#21858-6 "Grade Cancer"
+                SpecificFocusOfFinding value is type ref(CancerDisorderPresent)
+0..0			AnatomicalLocation
+0..0            Device
+
+
+EntryElement:   TumorPresent
+Based on:		BodyStructurePresent
+Concept:        SCT#395557000 "Tumor finding (finding)"
+Description:    "The presence of an abnormal mass of tissue (neoplasm) that results when cells divide more than they should or do not die when they should. Tumors may be benign (not cancer), or malignant (cancer). (source: NCI Dictionary)."
+                FindingTopicCode from TumorMorphologyVS
+// note - like other existential findings, the topic code is WHAT is present or absent, and the result code is present or absent (in this case, present)
+0..1            IsPrimaryTumor
+// TODO: Add tumor size
+
+
+        Element:        IsPrimaryTumor
+        Concept:        TBD
+        Description:    "Whether the tumor is the original or first tumor in the body, for a particular cancer."
+        Value:          CodeableConcept from YesNoUnknownVS    // could be boolean but this VS allows for unknown
+
+
+EntryElement:   CancerProgression
+Based on:       CodedNonLaboratoryObservation
+Description:    "The course of a disease, such as cancer, as it becomes worse or spreads in the body. (source: NCI Dictionary)"
+Concept:        SCT#246453008 "Course of illness (attribute)"
+                FindingResult from CancerProgressionVS
+                FindingTopicCode is SCT#246453008 "Course of illness (attribute)"
+                SpecificFocusOfFinding value is type ref(CancerDisorderPresent)
+// CancerProgressionEvidence is not yet part of M-CODE, but I don't see a better way to include it for FluxNotes
+0..*            CancerProgressionEvidence
+0..0			AnatomicalLocation
+0..0            Device	
+
+		Element:		CancerProgressionEvidence
+		Description:	"The type of evidence considered in determining disease progression."
+		Value:			CodeableConcept from CancerProgressionEvidenceVS	
+
+
+/**** replaced by Onset of CancerDisorderPresent 
```

Comment:
This is fine as long as we're clear to identify this as the earliest date of cancer diagnosis in documentation.  Noting this because the HL7 Patient Care team tends to associate "onset" with the first noticeable event that could have been related to the disease or condition.  For example, it is possible that the patient reported the onset of breast cancer through self-examination of a lump one month prior to a primary care physician visit.  The actual diagnosis date of breast cancer could have been then months later.